minos/networks/handlers/decorators.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (C) 2020 Clariteia SL
+#
+# This file is part of minos framework.
+#
+# Minos framework can not be copied and/or distributed without the express
+# permission of Clariteia SL.
+from functools import (
+    cache,
+)
+
+
+class BaseDecorator:
+    def __call__(self, fn):
+        def wrapper(*args, analyze_mode: bool = False, **kwargs):
+            if not analyze_mode:
```

Comment:
could you explain better the mean of "analyze_mode" argument? briefly not a detailed explaination